sale_stock_picking_blocking/i18n/sale_stock_picking_blocking_BACKUP_1781278.pot

```diff
@@ -0,0 +1,165 @@
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:ir.model.fields,field_description:sale_stock_picking_blocking.field_sale_delivery_block_reason__create_date
+msgid "Created on"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:ir.model.fields,field_description:sale_stock_picking_blocking.field_res_partner__default_delivery_block
+#: model:ir.model.fields,field_description:sale_stock_picking_blocking.field_res_users__default_delivery_block
+msgid "Default Delivery Block Reason"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:ir.model.fields,field_description:sale_stock_picking_blocking.field_sale_order__delivery_block_id
+#: model:ir.ui.menu,name:sale_stock_picking_blocking.menu_sale_stock_picking_blocking_reason
+msgid "Delivery Block Reason"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model_terms:ir.ui.view,arch_db:sale_stock_picking_blocking.view_sales_order_filter
+msgid "Delivery Blocked Sales"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:ir.model.fields,field_description:sale_stock_picking_blocking.field_sale_delivery_block_reason__description
+msgid "Description"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:ir.model.fields,field_description:sale_stock_picking_blocking.field_sale_delivery_block_reason__display_name
+msgid "Display Name"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:ir.model.fields,field_description:sale_stock_picking_blocking.field_sale_delivery_block_reason__id
+msgid "ID"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:ir.model.fields,field_description:sale_stock_picking_blocking.field_sale_delivery_block_reason____last_update
+msgid "Last Modified on"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:ir.model.fields,field_description:sale_stock_picking_blocking.field_sale_delivery_block_reason__write_uid
+msgid "Last Updated by"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:ir.model.fields,field_description:sale_stock_picking_blocking.field_sale_delivery_block_reason__write_date
+msgid "Last Updated on"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:ir.model.fields,field_description:sale_stock_picking_blocking.field_sale_delivery_block_reason__name
+msgid "Name"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model_terms:ir.ui.view,arch_db:sale_stock_picking_blocking.view_order_form
+msgid "Release Delivery Block"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:res.groups,name:sale_stock_picking_blocking.group_sale_delivery_block
+msgid "Release Delivery Block in Sales Orders"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+#: model:ir.actions.act_window,name:sale_stock_picking_blocking.action_sale_stock_picking_blocking_reason
+#: model:ir.model,name:sale_stock_picking_blocking.model_sale_delivery_block_reason
+msgid "Sale Delivery Block Reason"
+msgstr ""
+
+#. module: sale_stock_picking_blocking
+<<<<<<< HEAD
```

Comment:
@dreispt There are some rebasing errors here.

Reply:
Hmm, those are backup files left by the original author. I guess they can be removed.